portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="navbar">
+      <ul>
+        <li><a href="index.html">home</a></li> 
+        <li><a href="projects.html">projects</a></li> 
+        <li><a href="blog.html">blog</a></li> 
+        <li><a href="photos.html">photos</a></li> 
+        <li><a href="random.html">random</a></li>
+        </ul>
+    </div> 
+    
+    <div id="projects-content">
+      <h1>Projects and stuff I've worked on</h1>
+      <h4>Currently working as a STEP Intern @ Google remotely on the Play Analytics Team.</h4> 
+      <h2>Coding stuffs:</h2> 
+      <div class="project-block" id="avatar">
+        <h3>Avatar: The Last Infinite Runner</h3>
+        <h4>May 2020 | <a href="https://cyu6.github.io/avatar-runner" target="_blank">Play!</a> | <a href="https://github.com/cyu6/avatar-runner" target="_blank">GitHub</a></h4> 
+        <p>Infinite runner game based on the TV show "Avatar: the Last Airbender"</p>
+        <p>Used Three.js graphics library, JavaScript, HTML/CSS, Blender, Node.js</p> 
+      </div>
+      <div class="project-block" id="cdpush">
+        <h3>Config Cleanup @ Google</h3>
+        <h4>June - September 2019 | Engineering Practicum Intern</h4> 
+        <p>CDPush team in Sunnyvale. Developed a C++ server to scan and clean up files in a storage system.</p>
+        <p>Used C++, learned about Go, distributed systems, file management systems</p> 
+      </div>
+      <h2>I've recently become interested in UI/UX design:</h2>
+      <ul>
+        <li>MIT Beat the Pandemic II Project: <a href="">link to wireframe</a></li> 
+        <li>KoKo Pops Design Team Member: <a href="">link to work</a></li> 
```

Comment:
nit: i think for things that are not yet completed, even during development might be good practice to start adding TODOs. For the HTML, you can just add the <!-- TODO ..... add the links -->, etc.

Just looking ahead--not sure how this would work later with the capstone / github since TODOs can't be linked up with the buganizer, but there should be some equivalent that we can set-up later with issue #'s.

Reply:
edited this in the projects PR
will keep this in mind for the future!